plugins/europeana/thumbnail.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file Functions for working with the Europeana Thumbnail API
+ * @see https://pro.europeana.eu/resources/apis/record#thumbnails
+ */
+
+// TODO: remove this when the issue noted in the url plugin is resolved upstream
+import { URL } from '../url';
+
+export default function thumbnailUrl(uri, params = {}) {
+  const url = new URL('https://api.europeana.eu/api/v2/thumbnail-by-url.json');
+  for (const key of Object.keys(params)) {
+    url.searchParams.set(key, params[key]);
+  }
+  url.searchParams.set('uri', uri);
+  return url.toString();
+}
+
+export function thumbnailTypeForMimeType(mimeType) {
+  if (typeof mimeType === 'undefined') return null;
+  if (mimeType.startsWith('image/')) {
```

Comment:
Would using a switch statement here work?
It would be considered less complex and can have just one return statement at the end to get rid of the codeclimate warnings.